ee/vellum_ee/workflows/display/workflows/get_vellum_workflow_display_class.py

```diff
@@ -1,33 +1,45 @@
-from typing import Optional, Type
+import types
+from typing import TYPE_CHECKING, Optional, Type
 
 from vellum.workflows.types.generics import WorkflowType
-from vellum_ee.workflows.display.types import WorkflowDisplayContext, WorkflowDisplayType
+from vellum_ee.workflows.display.types import WorkflowDisplayContext
+from vellum_ee.workflows.display.utils.registry import get_from_workflow_display_registry
+
+if TYPE_CHECKING:
+    from vellum_ee.workflows.display.workflows import BaseWorkflowDisplay
+
+
+def _get_workflow_display_class(*, workflow_class: Type[WorkflowType]) -> Type["BaseWorkflowDisplay"]:
```

Comment:
This method now looks very similar to `get_node_display`. Part of me wants to move both methods to the `registry` module, but again, that would be a breaking change here unfortunately so not sure if it's worth it yet